### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DaphneMuller**
Suggestion for some simple instructions to setup a basic development setup for Collectives

Signed-off-by: Daphne Muller <86835268+DaphneMuller@users.noreply.github.com>

### 📝 Summary

* Resolves: # 

I was trying to setup a dev environment and saw a simple outline was missing. Here's a suggestion

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
B | A

n/a

### 🚧 TODO

- [ ] check if I got the nvm version correct

### 🏁 Checklist

- [ ] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [ ] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [ ] Tests (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [ ] Documentation ([README](https://github.com/nextcloud/collectives/blob/main/README.md) or [documentation](https://github.com/nextcloud/collectives/blob/main/docs/)) has been updated or is not required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,18 @@ for help in case of questions.
 
 Documentation for developers can be found at [DEVELOPING.md](DEVELOPING.md).
 
+## Development setup
+This app requires [Text](https://github.com/nextcloud/text) and [Circles](https://github.com/nextcloud/circles).
+You also need nvm v16.16.0 to compile the JavaScript. 
+After installing Text and Circles:
+
+1. Clone Collectives into the apps folder of your Nextcloud: 
+   `git clone https://github.com/nextcloud/circles`
```
Comment (juliushaertl) [inline README.md:43]:
```suggestion
   `git clone https://github.com/nextcloud/collectives`
```

Comment (DaphneMuller) [reply README.md:43]: jup embarrassing 🙈 thank you for the fast review :)

**Comment by DaphneMuller** [comment]
I'm lost in Github and now there are two files. I hope I'm not clicking wrong buttons all around.

File: README.md [hunk 2]
```diff
@@ -34,6 +34,18 @@ for help in case of questions.
 
 Documentation for developers can be found at [DEVELOPING.md](DEVELOPING.md).
 
+## Development setup
+This app requires [Text](https://github.com/nextcloud/text) and [Circles](https://github.com/nextcloud/circles).
+You also need nvm v16.16.0 to compile the JavaScript.
```
Comment (marcelklehr) [inline README.md:39]: You likely mean node v16 which can be installed using nvm ;)

**Comment by marcelklehr** [comment]
>  now there are two files

all good. there's two *commits* because you made two edits

File: README.md (hunk 2 above)
Comment (DaphneMuller) [reply README.md]: Thank you Marcel!! Added it to the file =)

**Comment by mejo-** [comment]
Thanks a lot @DaphneMuller :heart: 

To better understand your motivation behind this pull request: your idea is to provide a TL;DR of the setup instructions in `DEVELOPING.md`, right? Maybe it would be better to update `DEVELOPING.md` instead and simplify it a bit?

To be honest I'm a bit reluctant to have two separate places where instructions for developing are documented. In my experience this leads to outdated documentation in most cases :grimacing: 

I'm also fine with moving parts of `DEVELOPING.md` to `README.md` if deemed useful. I just try to avoid duplicated documentation.

**Comment by DaphneMuller** [comment]
thank you for your sweet and patient reply 🤗 I get your point about wanting to keep everything centralised in the developing.md file. Makes sense, I didn't realise that when adding the PR. I thought it would be nice to give people a quick way to get started because somehow I couldn't find this out quickly myself and then link to the larger developer docs but your way makes total sense, so I'm closing this thread. 

cheers!

**Pull request closed without merging**